src/components/helper-tool.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { CSSProperties } from "react";
+import { XMarkIcon } from '@heroicons/react/20/solid'
+
+type HelperToolProps = {
+  data: Record<string, any>;
+  locale?: string;
+};
+
+export const HelperTool = ({ data, locale }: HelperToolProps) => {
```

Comment:
Rename this and the file to something more meaningful. HelperTool means nothing from an outside perspective.